zh_cn.lang

```diff
@@ -16,57 +16,56 @@ gui.xaero_minimap=小地图
 gui.xaero_miscellaneous=杂项
 gui.xaero_edit_mode=编辑模式
 gui.xaero_instructions=教程/帮助
-gui.xaero_not_ingame=你必须先进入游戏开启编辑模式
+gui.xaero_not_ingame=你必须进入游戏才能开启编辑模式
 gui.xaero_welcome_edit_mode=欢迎使用编辑模式
-gui.xaero_howto_select=左键点击选择界面
-gui.xaero_howto_drag=左键点击并拖动界面
-gui.xaero_howto_deselect=右键取消选择界面
-gui.xaero_howto_center=C键固定界面居中
-gui.xaero_different_centered=某些界面有着不同的居中方式
-gui.xaero_howto_flip=F键翻转界面
-gui.xaero_howto_settings=S键快速访问选择的界面关联设置
-gui.xaero_howto_preset=点击<预设方案>选择一个预设方案
-gui.xaero_howto_save=点击<确认>保存更改
-gui.xaero_howto_cancel=点击<取消>放弃你的更改
+gui.xaero_howto_select=左键点击 - 选择界面
+gui.xaero_howto_drag=左键点击拖动 - 移动界面
+gui.xaero_howto_deselect=右键点击 - 取消选择界面
+gui.xaero_howto_center=C键 - 切换界面居中
+gui.xaero_different_centered=某些界面有着不同的居中样式
+gui.xaero_howto_flip=F键 - 切换界面翻转
+gui.xaero_howto_settings=S键 - 快速访问所选界面的设置
+gui.xaero_howto_preset=点击 <预设方案> 选择一个预设方案
+gui.xaero_howto_save=点击 <确认> 保存更改
+gui.xaero_howto_cancel=点击 <取消> 放弃更改
 gui.xaero_minimap_settings=小地图设置
 gui.xaero_miscellaneous_settings=杂项设置
 gui.xaero_notification_settings=提醒设置
 gui.xaero_better_pvp_settings=Better PvP 设置
-gui.xaero_server_disabled=服务器可能禁止了某些Mod特性
+gui.xaero_server_disabled=服务器可能禁止了某些Mod功能
 gui.xaero_waypoints=路径点
 gui.xaero_box_potion_effects=药水效果界面
 gui.xaero_box_armour_status=装备状态界面
-gui.xaero_box_sprint=一键冲锋
-gui.xaero_box_sneak=潜行开关
-gui.xaero_box_notifications=屏幕提醒，比如 饥饿/低血量/爆炸/弓箭.
-gui.xaero_box_xp=经验显示
+gui.xaero_box_sprint=一键疾跑
+gui.xaero_box_sneak=保持潜行
+gui.xaero_box_notifications=屏幕提醒, 比如 低血量/低饥饿值/低氧气值/爆炸危险/弓箭发射
```

Comment:
"gui.xaero_box_sneak" has adjusted text.
“gui.xaero_box_notifications” uses the official Chinese translation of mojang. 
In my actual test, even if the bow and arrow are not shot at the player, it will trigger a reminder, so I translate the part of the bow and arrow like this

Reply:
I know how arrow notification works.
This translates back as "reminder" here, which isn't correct but could make sense in your language, so I'm not sure. 
"Arrow launch" seems strange because that is not what is being detected. It detects flying arrows near you, not the launch moment.

Reply:
Have been adjusted.